### ISSUE TITLE
Make PhantomJS trust https with self-signed certificate

### ISSUE THREAD

**Issue by asolntsev** [codeborne/selenide#185]

**Comment by asolntsev** [comment]
This code should help:
```java
        capabilities.setCapability(PhantomJSDriverService.PHANTOMJS_CLI_ARGS,
            new String[] {"--web-security=no", "--ignore-ssl-errors=yes"});
```

**Comment by asolntsev** [comment]
Implemented in Selenide 2.19

**Comment by svpernova09** [comment]
Just wanted to say thanks for this PR, saved me a ton of headache finding `--ignore-ssl-errors=yes"`
